app/Http/Controllers/Auth/VerificationController.php

```diff
@@ -49,14 +49,14 @@ public function __construct()
 
     public function show(Request $request)
     {
-        $u = Auth::user();
-        if (is_object($u) && $u->hasVerifiedEmail()) {
-            // $u is logged in and verified user
-            return Inertia::render('Home/Index', ['user' => $u]);
+        $user = Auth::user();
+        if (is_object($user) && $user->hasVerifiedEmail()) {
+            // $user is logged in and verified user
+            return Inertia::render('Home/Index', ['user' => $user]);
```

Comment:
We should maybe think about using our `UserPresenter` to return this `$user` data to the front end so we get our keys converted and we have full control over what's ending up on the front end.  We'd need this in both branches of the if statement here

Reply:
:+1: I have rewritten this Controller a bit. I have finally also managed to pass a prop to say they just verified their email, so a message can be displayed on the dashboard (I think you would probably do that with flash messages? maybe we can move to that at a later point.) It can also sure do with some prettifying :D 

![email_verified](https://user-images.githubusercontent.com/1532276/91514475-c3961c80-e8de-11ea-85ff-0baa9b9ba926.png)